core/base/inc/TNamed.h

```diff
@@ -32,6 +32,8 @@ class TNamed : public TObject {
    TString   fName;            //object identifier
    TString   fTitle;           //object title
 
+   void SavePrimitiveNameTitle(std::ostream &out, const char *variable_name);
```

Comment:
```suggestion
   static void SavePrimitiveNameTitle(std::ostream &out, const char *variable_name);
```

Reply:
static does not work here

Reply:
why not?

Reply:
Then one need extra argument for `this` pointer

Reply:
My bad.  I commented on the wrong routine. :(